addons/l10n_es_edi_facturae/data/template/account.tax-es_common.csv

```diff
@@ -0,0 +1,52 @@
+"id","l10n_es_edi_facturae_tax_type"
+"account_tax_template_s_irpf19a","04"
+"account_tax_template_p_irpf19a","04"
+"account_tax_template_s_irpf195a","04"
+"account_tax_template_p_irpf195a","04"
+"account_tax_template_s_irpf20a","04"
+"account_tax_template_p_irpf20a","04"
+"account_tax_template_s_irpf21a","04"
+"account_tax_template_p_irpf21a","04"
+"account_tax_template_s_irpf21","04"
+"account_tax_template_p_irpf21p","04"
+"account_tax_template_p_irpf21t","04"
+"account_tax_template_p_irpf21td","04"
+"account_tax_template_p_irpf21te","04"
+"account_tax_template_s_irpf20","04"
+"account_tax_template_p_irpf20","04"
+"account_tax_template_s_irpf24","04"
+"account_tax_template_p_irpf24","04"
+"account_tax_template_s_irpf24_rdc","04"
+"account_tax_template_p_irpf24_rdc","04"
+"account_tax_template_p_irpf35cya","04"
+"account_tax_template_s_irpf15","04"
+"account_tax_template_p_irpf15","04"
+"account_tax_template_p_irpf15e","04"
+"account_tax_template_s_irpf18","04"
+"account_tax_template_p_irpf18","04"
+"account_tax_template_s_irpf19","04"
+"account_tax_template_p_irpf19","04"
+"account_tax_template_p_irpf19ca","04"
+"account_tax_template_p_irpf19cs","04"
+"account_tax_template_s_irpf9","04"
+"account_tax_template_p_irpf9","04"
+"account_tax_template_s_irpf7","04"
+"account_tax_template_p_irpf7","04"
+"account_tax_template_p_irpf7e","04"
+"account_tax_template_s_irpf2","04"
+"account_tax_template_p_irpf2","04"
+"account_tax_template_s_irpf1","04"
+"account_tax_template_p_irpf1","04"
+"account_tax_template_s_irpfnrue0","04"
+"account_tax_template_p_irpfnrue0p","04"
+"account_tax_template_s_irpfnrue19","04"
+"account_tax_template_p_irpfnrue19p","04"
+"account_tax_template_p_irpfnrnue0p","04"
+"account_tax_template_s_irpfnrnue0","04"
+"account_tax_template_s_irpfnrnue24","04"
+"account_tax_template_p_irpfnrnue24p","04"
+"account_tax_template_p_rp19","04"
+"account_tax_template_p_rrD19","04"
+"account_tax_template_s_gd0","05"
+"account_tax_template_p_dua_exempt","08"
+"account_tax_template_p_iva_isub","08"
```

Comment:
Not sure those need the type.  Already for purchase taxes, should we put it?  (it is not even used in the import of invoices somehow although it could be) 

It might be better to keep it empty for those, because it won't be in any file referring to this tax:

```suggestion
```